exercise-1.js

```diff
@@ -261,7 +371,8 @@
         //Remove all numbers that are less than 5.  Store the results in a new array.
 
         //your code...
```

Comment:
The exercise is asking to remove the numbers below 5 and store the results in a new array.